source/accessible-media/automatically-generated-captions.html.erb.md

```diff
@@ -58,10 +50,8 @@ YouTube's auto-caption service as an example.
 6.  [Upload the new caption file to
     YouTube](https://support.google.com/youtube/answer/2734796?hl=en).
```

Comment:
Three things: 

* Step 2 is done automatically, so why even bother listing it (“wait until automatic captions available”). The linked article says for auto captions: 
  > YouTube can use speech recognition technology to automatically create captions for your videos. If automatic captions are available, they'll automatically be published on the video. Learn how to review, edit, or unpublish automatic captions.
  >
  > Automatic captions are available in English, Dutch, French, German, Italian, Japanese, Korean, Portuguese, Russian, and Spanish.

* Steps 4 to 6 can be avoided by using the editor in YouTube’s interface. Shouldn’t that be our go-to option here?

* There are too many links in this sections that are all pointing away from the tuto… developer’s intro. I think we should briefly describe the step in our own words and not link out unless it is really necessary.